internal/extramanifest/policy.go

```diff
@@ -0,0 +1,125 @@
+package extramanifest
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"log"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	policyv1 "open-cluster-management.io/config-policy-controller/api/v1"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//+kubebuilder:rbac:groups=policy.open-cluster-management.io,resources=policies,verbs=get;list
+
+const ztpDeployWaveAnnotation = "ran.openshift.io/ztp-deploy-wave"
```

Comment:
A word about what `ztp-deploy-wave` is and why we sort the policies according to it? Google was not really helpful here

Reply:
A mechanism to order manifest application

Reply:
Currently, non image based install process uses this to order the policy enforcement as part of ZTP.